app/main/views/g_cloud.py

```diff
@@ -276,6 +276,25 @@ def search_services():
     )
 
 
+@direct_award.route('', methods=['GET'])
```

Comment:
Could this route be parameterised e.g. `/saved-searches/<framework_framework>`, and then that could be passed in to the template to avoid hard coding `framework_framework="g-cloud"` in the `url_for` calls?

Reply:
Briefs do something similar for suppliers view of opportunities:
```
/<framework_framework>/opportunities
/<framework_framework>/opportunities/<brief_id>
```